src/backend/aspen/workflows/nextstrain_run/run_nextstrain_ondemand.sh

```diff
@@ -14,13 +14,20 @@ df 1>&2
 cat /proc/meminfo 1>&2
 
 start_time=$(date +%s)
-build_date=$(date +%Y%m%d)
 
 aws configure set region $AWS_REGION
 
+if [ ! -z "${BOTO_ENDPOINT_URL}" ]; then
+  export aws="aws --endpoint-url ${BOTO_ENDPOINT_URL}"
+else
+  export aws="aws"
+fi
+
 # fetch aspen config
-genepi_config="$(aws secretsmanager get-secret-value --secret-id $GENEPI_CONFIG_SECRET_NAME --query SecretString --output text)"
+genepi_config="$($aws secretsmanager get-secret-value --secret-id $GENEPI_CONFIG_SECRET_NAME --query SecretString --output text)"
```

Comment:
We're using the $aws variable we set above with optional --endpoint-url flags to run aws cli commands.